### DISCUSSION TITLE
[Feature] Ordering in Shared Albums

### DISCUSSION THREAD

**Discussion by pig-cop** [immich-app/immich#4716] [Feature Request]
### The feature

It would be nice if the album ordering was available in the shared albums too.
It often happens that a family member adds photos to one of the shared albums but no one notices it if he doesn't say where he added the new photos.
The "most recent photo" sorting option would definitely solve this.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by OJ7** [comment]
To add on to this, having a timeline view within the album of when photos were added would be helpful (similar to the activity view in Google Photos albums).

**Comment by bo0tzz** [comment]
#3914
